code/style.css

```diff
@@ -1,5 +1,156 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+img {
+  font-size: 14px;
+}
 body {
-  background-color: lightblue;
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+  margin: 100px 0px;
+  background-color: rgb(252, 253, 254);
+}
+h1 {
+  display: left;
+  padding: 10px;  
+  margin-top: -100px;
+  margin-left: -200px;
+  float:left;   
+}
+h2 {
+  display: left;
+  padding: 10px;  
+  margin-top: -40px;
+  margin-left: -200px;
+  float:left;   
+}
+header {
+  background-image: url('pic1.jpg');
+  background-size: cover;
+  padding: 100px 0 100px 0;
+  text-align: center;
+}
+.header-text {
+  padding: 50px;
+  margin-top: 10px;
+  max-width: 500px;
+  margin: 0 auto;
+  color: white;
+}
+nav {
+  padding: 50px;
+  margin-top: 10px;
+  margin: auto;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+}
+.nav-link {
+  color: black;
+  text-decoration: none;
+  font-weight: bold;
+  padding: 10px 60px 0px 60px;
+  font-size: 15px;
+  box-sizing: border-box;
+}
+.intro {
+  background: darkgray;
+  padding: 20px;  
+  margin: 50px;  
+} 
+.intro-picture img{
+  size: 50%;
+  border-style: double;
+  padding:20px;
+  margin: 50px;
+  float: left;
+}
+.content-intro {
+  font-weight: 200;
+  max-width: 500px;
+  text-align: center;
+  padding-left: 50px;
+  margin-right: 100px;
+  margin-top: 150px; 
+  float: right; 
+}
+h3{
+  color: white;
+  font-weight: 200;
+  font-display: center;
+  background: rgb(124, 161, 91);
+  border: 20px;
+  padding: 10px;
+  margin-top: 450px; 
+  margin-bottom: 0px;
+  margin-left: 50px;
+  margin-right: 50px;
+}
+.interest-food {
+  width: 100%;
+  padding-left: 50px;
+  margin: auto;
+}
+.food-img img{
+  padding: 0px 50px ;
+  margin: 50px 30px;
+  float: right;
+}
+.food-text{
+  color: rgb(124, 161, 91);
+  display: block;
+  background-color: rgb(252, 248, 223);
+  max-width: 300px;
+  padding-left: 50px;
+  padding-right:300px;
+  padding-bottom: 30px;
+  padding-top: 30px;
+  margin-left: 50px;
+  margin-bottom:  0px;
+  float: left;
+}
+.interest-sport {
+  width: 0%;
+  padding-right: 50px;
+  margin: auto;
+}
+.sport-img img{
+  max-width: 250px;
+  padding: 50px;
+  margin: 50px 30px;
+  float: left;
+}
+.sport-text{
+  color: rgb(124, 161, 91);
+  display: block;
+  font-display: left;
+  background-color: rgb(251, 204, 211);
+  max-width: 300px;
+  padding-left: 300px;
+  padding-right: 50px;
+  padding-top: 30px;
+  padding-bottom: 30px;
```

Comment:
Here you can just write:
`padding: [top-value], [right-value], [bottom-value], [left-value];`
so:
`padding: 30px 50px 30px 300px;`